frontend/src/app/testResults/submissions/Submissions.scss

```diff
@@ -0,0 +1,42 @@
+.sr-showing-results-on-page {
```

Comment:
are we using this somewhere within the uploader context?

Reply:
we're also not using the other 2 class selectors in this file either I think

Reply:
Yeah why does this file exist 😕 nuked in [ffd6eef](https://github.com/CDCgov/prime-simplereport/pull/3766/commits/ffd6eef16767cc963125b8998aae13e2bd9fc04a)